testing/quilt/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=quilt
+pkgver=0.65
+pkgrel=0
+pkgdesc="Quilt allows you to easily manage large numbers of patches by keeping track of the changes each patch makes"
+url="http://savannah.nongnu.org/projects/quilt"
+arch="noarch"
+license="GPL"
+depends="bzip2 diffutils gawk gettext patch"
+makedepends="bash diffutils gawk gettext lynx perl procmail sed"
```

Comment:
`depends` are automatically installed when building the package, please don’t repeat them in `makedepends`.